frontend-nanodegree-arcade-game/js/app.js

```diff
@@ -302,7 +302,7 @@ const Life = class {
             player.increaseLife();
 
             //make life reappear in 5 seconds
-            //setTimeout(function() { this.reset(); }, 5000);
+            setTimeout(() => { this.reset(); }, 30000);
```

Comment:
If you use setInterval, it chooses a new random location every 30 seconds instead, which is also cool. We could have it so it changes the location of the heart every 5 seconds in the case no one picks it up. Try using that and let me know what you think. Also, your branch seems to be behind by a few commits, merge it with the latest version before submitting PR's please.

Reply:
I'll work on this today. When I made my PR my branch was up to date, however a couple of PRs got merged after I made my PR.